pkgs/games/steam/steam-udev.nix

```diff
@@ -0,0 +1,19 @@
+{ lib, stdenv, steamPackages }:
+
+stdenv.mkDerivation rec {
+  name= "steam-udev-${version}";
+  version = lib.getVersion steamPackages.steam.name;
+
+  src = steamPackages.steam.src;
+ 
+  installPhase = ''
+    install -d $out/lib/udev/rules.d
+    install -m644 lib/udev/rules.d/*.rules $out/lib/udev/rules.d
```

Comment:
Why not exposing the rules in the original `steamPackages.steam`?

Reply:
It could be added as udev rule package in a nixos module as well.

Reply:
If you say it's a good way to do it I will do a new pull request